pallets/msa/src/lib.rs

```diff
@@ -149,6 +153,27 @@ pub mod pallet {
 			Ok(())
 		}
 
+		#[pallet::weight(10_000)]
+		pub fn revoke_msa_delegate(
```

Comment:
since the delegate can be revoked by any of 2 parties. I think it might makes sense to have 3 params

```
origin: OriginFor<T>,
delegate_msa_id: MessageSenderId,
delegator_msa_id: MessageSenderId,
```
and the origin can be a key associated to delegate or delegator

Reply:
I think maybe we have a different function to make things a little easier. Here I do not know the submitter is the delegate or the delegator.

Maybe we have a `revoke_delegator`. I'll add an issue for this.